examples/hello-world/client/components/App.js

```diff
@@ -1,8 +1,27 @@
 import React from 'react'
+import { Helmet } from 'react-helmet'
+import Header from './Header.js'
 
 const RootComponent = () => (
   <div>
-    <h1>Hello, world!</h1>
+    <Helmet>
+      <link
+        rel="stylesheet"
+        href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
```

Comment:
Why are you using old bootstrap v3.3.7 ?

Reply:
I want to use the React-Bootstrap package (https://react-bootstrap.github.io/getting-started/introduction/). It has those links in the documentation because it's working with Bootstrap 3 components.

I think, that the Bootstrap 3 is enough for us :)

If you have any ideas how to improve this approach - you are welcome :)

Reply:
Looks like it's not necessary to use bootstrap components in this simple example.
Look at `create-react-app`.
Otherwise you can use bootstrap v4 with https://github.com/reactstrap/reactstrap.

Reply:
I'm sorry, what should I see at `create-react-app`?  I want to add style and header in all our examples, and they'll have the same style and architecture. It seems good to use the 'react-bootstrap' package because it looks very convenient for any front-end development. It has 12k stars, the `reactstrap` has 4k stars.

I don't know, maybe we'll change it in future, after first customer request. I can create the issue for customers-contributors ('replace blah-blah-blah), maybe it should be better for marketing.

Other guys, what're your thought? Change to `reactstrap`, leave the `react-bootstrap`, or maybe we should ask on twitter? :)